hawkbit-dmf/hawkbit-dmf-api/src/main/java/org/eclipse/hawkbit/dmf/json/model/DmfBatchDownloadAndUpdateRequest.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2022 Bosch Software Innovations GmbH and others.
```

Comment:
Should be
Copyright (c) 2022 Bosch.IO GmbH and others.

Reply:
done